metatron/src/main/resources/banner.txt

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
This is the most important part of this PR

Reply:
<3